dist/webinstaller/quick-lint-js-template.json

```diff
@@ -0,0 +1,103 @@
+{
+  "releases": [
+    {
+      "name": "linux.tar.gz",
+      "version": "${version}",
+      "lts": false,
+      "channel": "stable",
+      "date": "${releaseDate}",
+      "os": "linux",
+      "arch": "",
```

Comment:
Is an empty arch allowed? This is an x86_64/amd64 package.

Reply:
> I think it would be better if we add the arch: x86_64/amd64 to those empty values.

Yes, please do this.

Reply:
Done.